ts/components/ui/Markdown/index.tsx

```diff
@@ -50,18 +73,33 @@ body {
   padding: 0;
   color: ${customVariables.textColor};
   font-size: 16px;
+  font-family: 'Titillium Web';
+}
+
+ul, ol {
+  padding-left: 32px;
+}
+
+a {
+  border-radius: ${customVariables.borderRadiusBase}px;
+  font-weight: 600;
+  padding: 0 8px;
+  background-color: ${customVariables.textLinkColor};
+  box-shadow: inset 0 1px 0 1px ${customVariables.colorWhite};
+  color: ${customVariables.colorWhite};
+  text-decoration: none;
 }
 
 div.custom-block.io-demo-block {
   background-color: #c1f4f2;
-  border-radius: 8px;
-  margin-bottom: 50px;
-  padding: 10px;
+  border-radius: 4px;
+  margin-bottom: 32px;
+  padding: 4px 8px;
 }
 
 div.custom-block.io-demo-block .custom-block-body {
   position: relative;
-  padding-right: 50px;
+  padding-right: 48px;
```

Comment:
🌮 TIP: In generale, tendiamo ad usare una baseline di multipli di 8px quando possibile.